src/mongodb.js

```diff
@@ -3,9 +3,15 @@ var debug = require('debug')('feathers-sync');
 
 module.exports = function (config) {
   debug('setting up database %s', config.db);
-
+  var collection = config.collection || 'events';
   var client = mubsub(config.db);
-  var channel = client.channel(config.collection || 'events', config);
+
+  // Delete non-valid mubsub config options, causes a crash if
+  // these keys are present.
+  delete config.db;
```

Comment:
Rather than using `delete` we should use lodash's `omit`. So bring in `lodash.omit` and then do `omit(config, ['db', 'collection'])`. We don't want to mutate the config object in case people need it later. Good catch though @startupthekid!